React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,44 @@
+import { isArray } from "./utils";
+
+export function min(...args) {
+  if (args.length == 0) {
```

Comment:
Always use strict type comparison `===`. [Here](https://bytearcher.com/articles/equality-comparison-operator-javascript/) you'll find an interesting article that can help you understand better why it should be used. 😸